load.php

```diff
@@ -17,6 +17,7 @@
 
 define( 'PERFLAB_VERSION', '1.2.0' );
 define( 'PERFLAB_MAIN_FILE', __FILE__ );
+define( 'PERFLAB_MAIN_DIR_PATH', plugin_dir_path( PERFLAB_MAIN_FILE ) );
```

Comment:
```suggestion
define( 'PERFLAB_PLUGIN_PATH', plugin_dir_path( PERFLAB_MAIN_FILE ) );
```
`PERFLAB_MAIN_DIR_PATH` is not suitable name. Should we use `PERFLAB_PLUGIN_PATH` or `PERFLAB_PLUGIN_DIR_PATH`?

Reply:
Or `PERFLAB_BASE_PATH`?

Reply:
@felixarntz, I would be interested in hearing your thoughts on this.

Reply:
@mukeshpanchal27 @mehulkaklotar I would be on board with `PERFLAB_PLUGIN_PATH` or `PERFLAB_PLUGIN_DIR_PATH`. Maybe the latter is best since it should make very obvious what that is due to the similarity to the `plugin_dir_path()` function name.

Reply:
The constant is updated with `PERFLAB_PLUGIN_DIR_PATH`